features/admin.push-providers.v1/components/push-providers-grid.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ .push-provider-card-group {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    gap: 25px;
+    margin-bottom: 30px;
+    margin-left: 0.5em;
+ }
+ 
```

Comment:
Is a new line missing here?

Reply:
Fixed with [e7ff4f1](https://github.com/wso2/identity-apps/pull/7370/commits/e7ff4f1cf0e2a0ee3582d2977cd0f7ee1733c966)